### PR TITLE
docs: explanation of how to use the `session` filter to protect pages

### PR DESCRIPTION
**Pull request by datamweb**
Hello friends.
We have answered how to protect routes by several users in several discussions.
However, I recently received a similar message about how to protect the site panel.
I have covered this topic in this PR.

### CODE REVIEW

**Comment by datamweb** [comment]
Actually, I wanted to make the issue of routes protection more understandable by `changing the session filter`, but I think it is against this change. Therefore, I did not take any action.
```php 
if (url_is('login*') || url_is('/register')){
    return;
}
```
### Restrict Admin's Routes

For example, if you want to limit all routes leading to `localhost:8080/admin` and `localhost:8080/panel`, you need to add the following code in the `app\Config\Filters.php` file.

```php
public $filters = [
    'session' => [
        'before' => [
            'admin*', 
            'panel*',
        ]
    ]
];
```

File: docs/install.md [hunk 1]
```diff
@@ -140,8 +142,19 @@ use to protect your routes, `session`, `tokens`, and `chained`. The first two co
...
+These can be used in any of the [normal filter config settings](https://codeigniter4.github.io/CodeIgniter4/incoming/filters.html#globals), or [within the routes file](https://codeigniter.com/user_guide/incoming/routing.html#applying-filters).
 
-These filters are already loaded for you by the registrar class located at `src/Config/Registrar.php`.
+### Protect All Pages
+
+If you want to limit all routes (e.g. `localhost:8080/admin`, `localhost:8080/panel` and ...), you need to add the following code in the `app\Config\Filters.php` file.
+
+```php
+public $filters = [
+    'session' => ['except' => ['login*', '/register*']],
```
Comment (kenjis) [inline docs/install.md:153]:
```suggestion
    'session' => ['except' => ['login*', 'register*']],
```

Comment (datamweb) [reply docs/install.md]: Done!

File: docs/install.md [hunk 2]
```diff
@@ -140,8 +142,19 @@ use to protect your routes, `session`, `tokens`, and `chained`. The first two co
 to see if the user is logged in through either of authenticators, allowing a single API endpoint to
 work for both an SPA using session auth, and a mobile app using access tokens. The fourth, `auth-rates`,
 provides a good basis for rate limiting of auth-related routes.
+These can be used in any of the [normal filter config settings](https://codeigniter4.github.io/CodeIgniter4/incoming/filters.html#globals), or [within the routes file](https://codeigniter.com/user_guide/incoming/routing.html#applying-filters).
 
-These filters are already loaded for you by the registrar class located at `src/Config/Registrar.php`.
+### Protect All Pages
+
+If you want to limit all routes (e.g. `localhost:8080/admin`, `localhost:8080/panel` and ...), you need to add the following code in the `app\Config\Filters.php` file.
```
Comment (kenjis) [inline docs/install.md:149]: We use Unix style directory separators.

```suggestion
If you want to limit all routes (e.g. `localhost:8080/admin`, `localhost:8080/panel` and ...), you need to add the following code in the `app/Config/Filters.php` file.
```

File: docs/install.md [hunk 3]
```diff
@@ -140,8 +142,19 @@ use to protect your routes, `session`, `tokens`, and `chained`. The first two co
 to see if the user is logged in through either of authenticators, allowing a single API endpoint to
 work for both an SPA using session auth, and a mobile app using access tokens. The fourth, `auth-rates`,
 provides a good basis for rate limiting of auth-related routes.
+These can be used in any of the [normal filter config settings](https://codeigniter4.github.io/CodeIgniter4/incoming/filters.html#globals), or [within the routes file](https://codeigniter.com/user_guide/incoming/routing.html#applying-filters).
```
Comment (MGatner) [inline docs/install.md:145]: Not sure if you meant to link to two different versions of the UG?

**Review by MGatner — APPROVED** [review]

File: docs/install.md (hunk 3 above)
Comment (MGatner) [reply docs/install.md:145]: Sorry to be unclear! codeigniter4.github.io is the latest dev build of the UG and codeigniter.com is the public stable one. You used both versions- I wasn't sure if there was something in Filters on the dev version you wanted, or if it was simply a copy-paste mistake.

Comment (kenjis) [reply docs/install.md:145]: `https://codeigniter.com/user_guide/` is the official one for the stable release, and it is used in the existing docs.
Probably we should use it.

`https://codeigniter4.github.io/CodeIgniter4/` is the current UG of `develop` branch.
`https://codeigniter4.github.io/userguide/` is the same as `https://codeigniter.com/user_guide/`, but hosted by GitHub Pages.

Comment (datamweb) [reply docs/install.md]: Oh! @MGatner , as you said, it was a copy-paste mistake.
Thank you Kenjis.

**Done.**

**Review by MGatner — APPROVED** [review]
LGTM!

**Review by kenjis — APPROVED** [review]

**Pull request merged**